buildenv/jenkins/JenkinsfileBase

```diff
@@ -315,7 +319,7 @@ def setupParallelEnv() {
 		if (create_jobs) {
 			parallel create_jobs
 		}
-
+		return NUM_LIST
```

Comment:
The variable `NUM_LIST` is not visible here (it's confined to the scope where it's declared, which ends on line 265); our test builds are failing as a result:
```
00:15:35.923  Exception: groovy.lang.MissingPropertyException: No such property: NUM_LIST for class: Script1
```